code/__HELPERS/names.dm

```diff
@@ -103,24 +163,24 @@ GLOBAL_VAR(command_name)
 	var/name = ""
 
 	// Prefix
-	name += pick("Clandestine", "Prima", "Blue", "Zero-G", "Max", "Blasto", "Waffle", "North", "Omni", "Newton", "Cyber", "Bonk", "Gene", "Gib")
+	name += pick(GLOB.syndicate_prefix)

```

Comment:
Could you do `var/list/name = list()`
instead of `var/name = ""`
and end the proc with `return name.Join()`
instead of `return name` ?